src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -29,12 +70,12 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 	}
 
 	if (!owner || !repo || !sourceBranchName || !newBranchName) {
-		res.status(400).json(errorMessages[400]);
+		res.status(400).json(getErrorMessages(400));
```

Comment:
Correct me if I'm wrong, but `getErrorMessages` returns a string, right? Why do we call `.json()` then?

Reply:
Updated to `send`

Reply:
Shouldn't we return json though?...

Reply:
I don't think that makes much of a difference though, its only returning the messages which are rendered directly in the frontend.

Reply:
Well, in other places we do return JSONs, e.g. http://github.com/atlassian/github-for-jira/blob/fcad689efce720aa95309dba6e6b0a6b63057a8e/src/routes/api/jira/api-jira-verify-post.ts#L25-L25 . I think this is just the way we prefer to do error handling? I'd prefer to keep this uniformity.

Reply:
Or here http://github.com/atlassian/github-for-jira/blob/fcad689efce720aa95309dba6e6b0a6b63057a8e/src/routes/github/setup/github-setup-post.ts#L15-L15

Reply:
Updated to `json` and returned as `{ error: "" }`